composer.json

```diff
@@ -9,6 +9,7 @@
     },
     "require-dev": {
         "brain/monkey": "^2",
+        "mockery/mockery": "^1.2.4",
```

Comment:
This allows mocking entire classes, like `Loader`. It doesn't look like [Brain Monkey](https://brain-wp.github.io/BrainMonkey/docs/wordpress-hooks-added.html) can mock classes.